index.html

```diff
@@ -17,15 +17,18 @@ <h1>What's for Dinner?</h1>
     <section id="form-box" class="containers">
       <form id="type-form">
         <h2 class="form">What are you looking for?<span>*</span></h2>
-        <label for="side"><input type="radio" name="meal-type" id="side" value="side" class="form">Side</label>
-        <label for="main-dish"><input type="radio" name="meal-type" id="main-dish" value="main-dish" class="form"/>Main Dish</label>
-        <label for="dessert"><input type="radio" name="meal-type" id="dessert" value="dessert" class="form"/>Dessert</label>
-        <label for="entire-meal"><input type="radio" name="meal-type" id="entire-meal" value="entire-meal" class="form"/>Entire Meal</label>
-        <input type="submit" id="submit-button" value="LET'S COOK!" class="form"/>
+        <label for="side"><input type="radio" name="meal-type" id="side" value="side" class="form"> Side</label>
+        <label for="main-dish"><input type="radio" name="meal-type" id="main-dish" value="main-dish" class="form"/> Main Dish</label>
+        <label for="dessert"><input type="radio" name="meal-type" id="dessert" value="dessert" class="form"/> Dessert</label>
+        <label for="entire-meal"><input type="radio" name="meal-type" id="entire-meal" value="entire-meal" class="form"/> Entire Meal</label>
+        <button type="submit" id="submit-button" value="LET'S COOK!" class="form">LET'S COOK!</button>
       </form>
     </section>
     <section id="random-dish" class="containers">
-      <img src="assets/cookpot.svg" atl="cookpot icon">
+      <img class="" id="cookpot-icon" src="assets/cookpot.svg" atl="cookpot icon">
```

Comment:
Missed this during my review, but same comment here re: an empty string for a class. You can just delete `class=""` unless you plan to add a class here.